dialogue-apache-hc4-client/src/main/java/com/palantir/dialogue/hc4/ApacheHttpClientChannels.java

```diff
@@ -138,8 +140,7 @@ public static CloseableClient createCloseableHttpClient(ClientConfiguration conf
         connectionManager.setDefaultMaxPerRoute(Integer.MAX_VALUE);
         // Increased from two seconds to twenty-five seconds because we have strong support for retries
         // and can optimistically avoid expensive connection checks.
```

Comment:
javadoc now out of date - shall we put a little sentence explaining the choice of 55 (over say 1 minute) and the 2.5 divisor?

Reply:
:+1: